examples/flux_integration/simple_fnn/fit_step_function.jl

```diff
@@ -0,0 +1,117 @@
+# Set up the environment to run this example. Make sure you're within the folder that this
+# file lives in.
+using Pkg
+Pkg.activate(@__DIR__)
+Pkg.instantiate()
+
+
+using Plots, Random, Stheno, Flux, Zygote
+rng = MersenneTwister(5)
+T = Float64
+
+
+# define the step function 
+# step_func(x) = 0.0 if x<=0, 1.0 if x>0
+step_func(x) = (ϵ=T(0.01)*randn(rng, T); x>T(0.0) ? T(1.0)+ϵ : T(0.0)+ϵ)
```

Comment:
It's a little hard to read this. Consider
```suggestion
function step_func(x)
    ϵ = T(0.01) * randn(rng, T)
    return x > zero(T) ? one(T) + ϵ : zero(T) + ϵ
end
```